widgets/src/main/java/org/pentaho/mantle/client/dialogs/folderchooser/SelectFolderDialog.java

```diff
@@ -36,6 +36,8 @@
 import com.google.gwt.user.client.ui.TreeItem;
 import com.google.gwt.user.client.ui.VerticalPanel;
 
+import static org.pentaho.mantle.client.dialogs.folderchooser.FolderTree.getDefaultSaveLocation;
```

Comment:
I think this is counter to our coding style - you have to avoid static imports except in the case of unit tests.

Reply:
Ok, I will call it directly using FolderTree.getDefaultSaveLocation()